### PR TITLE
Update sponsoring-an-open-source-contributor.md

### PR DESCRIPTION
**Pull request by rpshbhandari**
### Why:
Helps the user understand organisational profile and user profile better.
Closes [issue link]

### Check off the following:

- [x ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@RupeshBhandari Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ If the account you want to sponsor does not have a profile on {% data variables.
 
 Before you can sponsor an account, you must have a verified email address. For more information, see "[Verifying your email address](/github/getting-started-with-github/verifying-your-email-address)."
 
-1. On {% data variables.product.product_name %}, navigate to the profile of the user or organization you want to sponsor.
+1. On {% data variables.product.product_name %}, navigate to the profile of the (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile) or (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-organizations-profile) you want to sponsor.
```
Comment (ajs256) [inline content/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor.md:55]: I don't think this is valid Markdown link syntax, is this what you meant?
```suggestion
1. On {% data variables.product.product_name %}, navigate to the profile of the (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile)[user] or (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-organizations-profile)[organization] you want to sponsor.
```

**Review by ajsirota — CHANGES_REQUESTED** [review]

File: content/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor.md (hunk 1 above)
Comment (RupeshBhandari) [reply content/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor.md:55]: Yeah thats what is what i meant. Thanks

**Review by rpshbhandari — COMMENTED** [review]
Looks good!

**Comment by lecoursen** [comment]
Thanks for taking the time to open this @RupeshBhandari, but we expect folks to already understand the differences between users and organizations here. Adding the links could be distracting, so I'm going to close this PR. Thanks again for your interest in our docs!

**Pull request closed without merging**